src/Service/Unzer.php

```diff
@@ -488,7 +488,7 @@ public function getBankDataFromCharge(Charge $charge): string
 
     public function getPaymentProcedure(string $paymentMethod): string
     {
-        if (in_array($paymentMethod, ['paypal', 'card', 'installment-secured', 'applepay', 'paylater-installment'])) {
+        if (in_array($paymentMethod, ['paypal', 'card', 'applepay', 'installment-secured', 'paylater-installment'])) {
```

Comment:
interesting that the order of the payment ids changes the behaviour of this method, why this?